packages/perseus/src/components/fixed-to-responsive.tsx

```diff
@@ -70,13 +75,13 @@ const FixedToResponsive: any = createReactClass({
         } else {
             this._cacheViewportSize();
         }
-    },
+    }
 
-    componentWillUnmount: function () {
+    componentWillUnmount() {
```

Comment:
I imagine these methods aren't typed because they're implementing the component interface which types them?

Reply:
Well, `componentWillUnmount` doesn't take any parameters and doesn't return anything so this feels accurate. :) 

https://react.dev/reference/react/Component#componentwillunmount